bin/apt-dependencies.sh

```diff
@@ -126,13 +125,13 @@ mkdir -p /usr/share/lintian/profiles/couchdb
 chmod 0755 /usr/share/lintian/profiles/couchdb
 if [[ ${VERSION_CODENAME} =~ ${debians} ]]; then
   cp ${SCRIPTPATH}/../files/debian.profile /usr/share/lintian/profiles/couchdb/main.profile
-  if [ ${VERSION_CODENAME} == "jessie" ]; then
+  if [ "${VERSION_CODENAME}" == "jessie" ]; then
```

Comment:
@nickva Should we completely remove Debian Jessi support here?

Reply:
Yeah, good idea to clean it up

Reply:
ok, removed.

Reply:
Thanks, Ronny